libs/com.etendorx.das_core/src/main/java/com/etendorx/entities/mapper/lib/DTOConverter.java

```diff
@@ -46,11 +43,6 @@ public interface DTOConverter<E, F, G> {
    * @param entities
    * @return a list of DTOs converted
    */
-  default Iterable<F> convert(Iterable<E> entities) {
-    List<F> dtos = new ArrayList<>();
-    for (E entity : entities) {
-      dtos.add(convert(entity));
-    }
-    return dtos;
-  }
+  Iterable<F> convert(Iterable<E> entities);
```

Comment:
**Suggestion:** The convert method in the DTOConverter interface has been changed from a default method to an abstract method. This might break existing implementations of the interface. Consider reverting this change if it's not necessary.
```suggestion
  default Iterable<F> convert(Iterable<E> entities) {
    List<F> dtos = new ArrayList<>();
    for (E entity : entities) {
      dtos.add(convert(entity));
    }
    return dtos;
  }
```